packages/expo-web-browser/src/WebBrowser.types.ts

```diff
@@ -19,21 +19,21 @@ export type OpenBrowserOptions = {
   windowFeatures?: string;
 };
 
-export type AuthSessionResult = RedirectResult | BrowserResult;
+export type WebBrowserAuthSessionResult = WebBrowserRedirectResult | WebBrowserResult;
 
-export type CustomTabsBrowsersResults = {
+export type WebBrowserCustomTabsResults = {
   defaultBrowserPackage?: string;
   preferredBrowserPackage?: string;
   browserPackages: string[];
   servicePackages: string[];
 };
 
-export type BrowserResult = {
+export type WebBrowserResult = {
   // cancel and dismiss are iOS only, opened is Android only
   type: 'cancel' | 'dismiss' | 'opened';
 };
```

Comment:
Let's change it and go with sth like this [TS Playground](https://www.typescriptlang.org/play/index.html#code/KYDwDg9gTgLgBAYwgOwM7wOrAEYCEoQDuqwUASsKgK4A2MAKgJ5jBwC8cA3gFBxwD0AKkG8+guAEsA8gGU4KGo1FxB-UQGEAggDl1AUQAyALjgByBAENkCYDVMAaUUJF8Vk2fOSLlq0QBEASRkAWSCZE1MAEwlUAFsY1AcnYR84TWRIgglIz29XXz4pAAU9bT0-CIgWZGBIpIBfOAtURBR0bm5QSFg4GGZWLDwCYlIKajomFnZe-ogAMzhB-CISckpaBn6AbQBrYEZ5mZZDpeHVsY3J4ABdDq7oeD6p05XR9bppnj4n4BMXkbW402LAA3Nx6iCgA)